axios/index.tsx

```diff
@@ -0,0 +1,3 @@
+import axios from 'axios';
+
+export default axios.create({ baseURL: "https://localhost/4000" });
```

Comment:
I've already setup an `isProduction` config variable in `next.config.js`, you could set up the server url in that file by checking `isProduction` and export that url via webpack, that should be a better way to do this, also the url would be `http://localhost:4000` if :4000 were being used by the backend. The webpack idea is how I'd suggest you do it, let us know if you have something better in mind